sdk/storage/Azure.Storage.Files.DataLake/src/DataLakeFileSystemClient.cs

```diff
@@ -576,6 +579,7 @@ private void SetNameFieldsIfNull()
         public virtual Response<FileSystemInfo> Create(
             Models.PublicAccessType publicAccessType = Models.PublicAccessType.None,
             Metadata metadata = default,
+            DataLakeFileSystemEncryptionScopeOptions encryptionScopeOptions = default,
```

Comment:
Should we create `DataLakeFileSystemCreateOptions` and pack all the optional parameters inside?
Current approach won't scale well if more optional concepts are introduced here.

I can see it's a match for how blobs got implemented, I'm not sure if that's right model to follow. If we're unsure we should get more eyes on this problem (i.e. discuss in review meeting).

Reply:
I agree, I would like to introduce `DataLakeFileSystemCreateOptions`.  Wanted to get some feedback first before implementing.  Will do.

Reply:
fixed.